docs/reference/juju-cli/list-of-juju-cli-commands/add-unit.md

```diff
@@ -11,7 +11,7 @@ Adds one or more units to a deployed application.
 ### Options
```

Comment:
The CLI docs should be autogenerated in `main`, so we should just defer to what's in `main` when you run `make run` or `make html` in `juju/docs`.